azure-pipelines.yml

```diff
@@ -0,0 +1,82 @@
+name: $(TeamProject)_$(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+trigger:
```

Comment:
I think it's better to build all branches
```suggestion
trigger:
  batch: true
  branches:
    include:
      - "*"

pr: none
```

Reply:
Suggestion committed

Reply:
Unneeded 'master' line removed